Assets/Scripts/EOSLobbyManager.cs

```diff
@@ -1696,6 +1699,75 @@ private void OnRTCRoomUpdateReceivingCompleted(ref UpdateReceivingCallbackInfo d
             }
         }
 
+        /// <summary>
+        /// Blocks the Local Lobby Member from receiving/sending audio from/to a Target Lobby Member. Only affects this Local Member. Does not affect other Lobby Members.
+        /// </summary>
+        /// <param name="status">To block or to unblock? True to Block, False to Unblock</param>
+        /// <param name="targetUserId">The Target Member's ProductUserID</param>
+        /// <param name="blockRTCParticipantComplete">Callback when BlockTargetRTCParticipant is completed</param>
+        public void BlockTargetRTCParticipant(bool status, ProductUserId targetUserId, OnLobbyCallback blockRTCParticipantComplete)
```

Comment:
A minor nit-pick on this: 
I would change this to `UpdateBlockStatusForRTCParticipant`, and flip the order of the parameters such that it's `('ProductUserId targetUserID, bool status, OnLobbyCallback blockRTCParticipantComplete)`

Reply:
Changes as requested have been made - so I am marking this as resolving in order to merge.